source/documentation/getting_started/get_started.erb

```diff
@@ -16,6 +16,24 @@ Once you have a personal account, you can access the GOV.UK PaaS admin tool for
 
 Contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk) if you have any questions.
 
+### Trial and paid accounts
+
+When your department, agency or team first [signs up to the GOV.UK PaaS](https://www.cloud.service.gov.uk/signup), their account is a free trial account.
+
+With a trial account, your department, agency or team can:
+
+- manage [users](/orgs_spaces_users.html#users-and-user-roles)
+- create and manage [spaces](/orgs_spaces_users.html#spaces)
+- deploy and run [apps](/deploying_apps.html#deploying-apps)
+- use [backing service](/deploying_services) free plans
+
+However, your department, agency or team has a limited [quota](/managing_apps.html#quotas) of resources, and cannot:
+
+- set up [custom domains](/deploying_services/use_a_custom_domain/#managing-custom-domains-using-the-cdn-route-service)
+- run [production apps](/deploying_apps.html#deploy-an-app-to-production) on the GOV.UK PaaS
+
+To set up custom domains and run production apps, your department, agency or team must have a paid account with the GOV.UK PaaS. To upgrade your department, agency or team's account from trial to paid, contact us by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
```

Comment:
This is where we should be signposting to a clearer upgrade flow

This could be supported in the tech docs by "before you start" content as suggested by Debs here: https://s3.amazonaws.com/prod.tracker2/resource/97196840/paas-mou-journey.pdf?response-content-disposition=inline&response-content-type=application%2Fpdf&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAJJBSFJ4TCVKKGAIA%2F20191014%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20191014T105503Z&X-Amz-Expires=1800&X-Amz-SignedHeaders=host&X-Amz-Signature=bf5566b6ea041e55d266b0cab8b78ee43fe493fa66c1db868e9ad3ba11078cd3

I have captured user needs for upgrade journey in this story: 
https://www.pivotaltracker.com/story/show/168090316

Reply:
@lukemalchergds I cannot open that link, can you advise?

Reply:
output from this story: https://www.pivotaltracker.com/story/show/166036324

Reply:
Agreed with @lukemalchergds that this does not stop this story from progressing